### DISCUSSION TITLE
Can't signup for free trial, it says i'm in, but then it doesn't inside my account settings.

### DISCUSSION THREAD

**Discussion by patrickcurl** [community/community#19104] [Copilot Conversations]
I keep getting looped back to the signup for free trial page, and keep getting sent in loops. 

It says I'm signed up, but if I go to my account settings and click on copilot it says I'm not but that I can signup for the free trial.

**Comment by MarcoPNS** [comment]
I have the same problem. I was in the test phase before but now I cant use it.

**Reply by davecheney** [reply]
@MarcoPNS there is a problem with your billing status. Please contact support and ask them to reset your billing status.

**Comment by D7EAD** [comment]
Hi!

If you're having issues activating the Github Copilot free 60-day trial on your account, I'd recommend raising a ticket with Github's Support Engineers. You can do so at [this](https://support.github.com/contact/account) link.
